src/Data/Impact.elm

```diff
@@ -418,10 +421,44 @@ getAggregatedScoreOutOf100 { trigram } impactsPerKg =
                 -- See the documentation at https://fabrique-numerique.gitbook.io/ecobalyse/alimentaire/impacts-consideres/score-100
                 (ln 2077 - ln value) / ln 2 * 20
            )
-        |> round
+        |> floor
         |> clamp 0 100
 
 
+foodCategories : Dict String { name : String, bounds : ( Int, Int ) }
+foodCategories =
+    Dict.fromList
+        [ ( "meats", { name = "Viandes", bounds = ( 500, 4000 ) } )
+        , ( "fruitsAndVegetables", { name = "Fruits et légumes", bounds = ( 30, 450 ) } )
+        , ( "cakes", { name = "Gâteaux", bounds = ( 100, 700 ) } )
+        ]
+
+
+getAggregatedCategoryScoreOutOf100 : Definition -> String -> Impacts -> Int
+getAggregatedCategoryScoreOutOf100 { trigram } foodCategory impactsPerKg =
+    case Dict.get foodCategory foodCategories of
+        Just { bounds } ->
+            let
+                ln =
+                    logBase e
+
+                ( min, max ) =
+                    bounds
+            in
+            impactsPerKg
+                |> getImpact trigram
+                |> Unit.impactToFloat
+                |> (\value ->
+                        -- See docs at https://fabrique-numerique.gitbook.io/ecobalyse/alimentaire/impacts-consideres/score-100#projet-declinaisons-du-score-100
+                        (ln (toFloat max) - ln value) / ln (toFloat max / toFloat min) * 20 * 5
+                   )
+                |> floor
+                |> clamp 0 100
+
+        Nothing ->
+            -99
```

Comment:
This makes me kinda uncomfortable... I understand it's an "impossible value"... would we want to use `0` which is still in the range of "acceptable scores", or instead return a maybe or a result?

Not sure about this, just a feeling, a "smell"

Reply:
![image](https://user-images.githubusercontent.com/41547/215525986-64df29c5-7dad-4d67-bee8-9c5ddc2e3d8b.png)

Fixed in https://github.com/MTES-MCT/ecobalyse/pull/210/commits/d975ddbab2add9dc2d117ac03c3ecccea0db959e